devel/bamtools/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        pezmaster31 bamtools 2.5.1 v
+
+categories          devel
+license             MIT
+platforms           darwin
+maintainers         nomaintainer
+
+description         a programmer's API and an end-user's toolkit for handling BAM files
+long_description    BamTools provides both {*}${description}.
+
+checksums           rmd160  a537678aaad8ea323f7d8f86d92983353220d0fc \
+                    sha256  a933067515bd36ee6edea2379bc156a38d54c67e4c11243eea6cb56a1ebb1ae3 \
+                    size    549902
+
+depends_lib-append  port:jsoncpp
```

Comment:
You should write this dependency in `path:` style so that jsoncpp-devel could satisfy it.

Reply:
Ok, do you mean something like:

```tcl
depends_lib-append          path:lib/libjsoncpp.dylib
```

I've never used the `path:` style before. To satisfy both, I assume path to the lib would do, with an ending `:jsoncpp`?

Reply:
`path:lib/libjsoncpp.dylib:jsoncpp`

Installed `jsoncpp-devel` first, and then installed this, and it was ok.